tasks/main.yml

```diff
@@ -292,7 +278,101 @@
       debug:
         var: __mssql_sqlcmd_input_file.stdout_lines
 
-    - name: Remove the {{ __mssql_sql_tempfile.path }} file
+    - name: Remove the tempfile
       file:
         path: "{{ __mssql_sql_tempfile.path }}"
         state: absent
+
+- name: Configure the sqlagent setting
+  include_tasks: mssql_conf_setting.yml
+  vars:
+    __mssql_conf_setting: "sqlagent enabled"
+    __mssql_conf_setting_value: "{{ mssql_enable_sql_agent }}"
+  when: mssql_enable_sql_agent is not none
+
+- name: Install the {{ __mssql_server_fts_packages }} package
```

Comment:
I would suggest to use `Ensure the MS SQL FTS packages` for the `name`

Reply:
Will do.

Reply:
Done